std_msgs/Header header 
float64[] range
float64[] bearing
float64[] inclination
float64 max_height_diff
uint32 num_neighbors
